curator/actions.py

```diff
@@ -1880,7 +1880,7 @@ def qualify_single_node(self):
             raise ActionError('Node "{0}" is not usable as a shrink node'.format(self.shrink_node))
         if not single_data_path(self.client, node_id):
             raise ActionError(
-                'Node "{0}" has multiple data paths and cannot be used '
+                'Node "{0}" has multiple data paths and cannot be used '.format(self.shrink_node) + \
```

Comment:
Please put it on the last line before the closing parenthesis.  You don't need to use the `+ \` syntax that way, and it should work just fine.

Reply:
If not, I stand corrected.

Reply:
Confirmed that it will work that way (in `ipython`):
```
In [2]: print(
   ...:     'a-'
   ...:     'b-{0}-'
   ...:     'c'.format('FOO')
   ...: )
a-b-FOO-c
```